### PR TITLE
HTML/CSS Project #6:  Building with Responsive Design

### PR DESCRIPTION
**Pull request by VanessaAoki**
We build a clone of thenextweb.com's homepage using HTML and CSS, float, grid, flexbox, and media query. This page works best using the firefox or google chrome browser, assets might look out of place using other browsers or resolutions.

The focus of this branch was to create a responsive site, which should have 3 major breaking points: the first one is supposed to be for mobile displays (max-width: 768px); the second one is for tablets displays (min-width: 768px); and the last one is for laptops/monitors (min-width: 1024px);

### CODE REVIEW

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## Hi @VanessaAoki and @realisticattorney 

You've done a good job :clap:  but there's a little more to be done before your project can be approved.

### [REQUIRED]

- [x] The screenshot in your README is not showing.

- [x] Increase the height of the top 3 articles

- [x] In the latest news section, you should add some margin/padding to the description and author/time

- [x] Reduce the height of the images in the **Hard Fork** section.

- [x] Your latest deals section should look exactly like the one [here](https://thenextweb.com/)

- [x] Your footer shouldn't be stacked on large and medium screen. Please follow the design [here](https://thenextweb.com/)

- [x] Please include the first navbar

### Make the required changes and submit for another review


## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Review by PhillipUg — APPROVED** [review]
# Status: Project Approved! 🥇 
Congratulations on making all the requested changes and getting this project approved. :+1:

- [OPTIONAL]: Consider adding a dark filter on the images in the `top section` and the `latest funding rounds... section`. If you observe the real page keenly, you will notice the dark filter from bottom-to-top. This is meant to make the words overlaying the images easier to read. 

You can merge this branch into the main branch when you're ready and go on to your next project.
:bulb:  Do not forget to submit your project completion form after merging this branch.

All the best team,

For any assistance, you can contact me on slack as **Phillip Musiime**

**Pull request merged**
